app/lib/utils/splitIntoColumns.ts

```diff
@@ -0,0 +1,21 @@
+export const splitIntoColumns = (items: any[], columnsCount: number) => {
+  const columns: any[][] = Array.from({ length: columnsCount }, () => []);
+  const rowsCount = Math.ceil(items.length / columnsCount);
```

Comment:
```suggestion
export const splitIntoColumns =<T,>(items: T[], columnsCount: number): T[][] => {
  const columns: T[][] = Array.from({ length: columnsCount }, () => []);
  const rowsCount = Math.ceil(items.length / columnsCount);
```
can we remove any like this?